src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -305,9 +333,18 @@ other Axom developers to use during development, in Axom GitLab CI testing, etc.
 
 
 #. **Install TPLs on all required LC machines.**
-   This step needs to be run on each of the machines named in Axom's standard host-configs.
-   When you are confident that everything is correct, become the service user
-   ``atk`` via the following command::
+
+   When you are confident that everything is correct and working, you will need
+   to perform this step on each of the machines named in Axom's standard host-configs.
+
+   .. important:: To install TPL builds to be shared by all Axom developers and used 
+                  in our GitLab CI, you will need to become the Axom service user ``atk``.
+                  There is a clone of the Axom repo in the ``/usr/workspace/atk/axom_repo``
+                  directory. After becoming ``atk``, you can go into that directory and
+                  switch to the branch you made to test your changes. Before running the
+                  TPL builds, make sure the branch is updated, including all submodules.
```

Comment:
Should we mention that this is only applicable to the subset of axom developers who can become `atk` ?

Reply:
I think that's implied since the /usr/workspace/atk directory is not accessible if you do not `xsu atk` and only members of the team can do that.  

The main point I was trying to make is that you really need to become atk and use that clone of the repo to build and install the shared TPLs.